frontend/package.json

```diff
@@ -19,33 +19,37 @@
     "start-with-backend": "HOST='http://localhost:3000' DISABLE_MIRAGE=true ember s"
   },
   "engines": {
-    "node": "^10"
+    "node": "8.* || >= 10.*"
```

Comment:
@allthesignals @trbmcginnis did the Ember Upgrade require this specific Node version config?

Reply:
Oops, good catch. @trbmcginnis would know, but it probably was an automated thing. We should keep it at 10.